src/backend/utils/misc/faultinjector.c

```diff
@@ -358,7 +412,9 @@ FaultInjector_InjectFaultIfSet_out_of_line(
 		if (strlen(entryShared->tableName) > 0 && strcmp(entryShared->tableName, tableNameLocal) != 0)
 			/* fault injection is not set for the specified table name */
 			break;
-
+		if (entryShared->nestingLevel !=0 && entryShared->nestingLevel != nestingLevel)
+		    /* fault injection is set for another nesting level */
+			break;
```

Comment:
Let's add empty lines before and after the check, as it is already done for the surrounding existing checks.
```suggestion

		if (entryShared->nestingLevel !=0 && entryShared->nestingLevel != nestingLevel)
		    /* fault injection is set for another nesting level */
			break;
			
```

Reply:
Fixed